catboost/private/libs/app_helpers/bind_options.cpp

```diff
@@ -76,6 +76,10 @@ void BindQuantizerPoolLoadParams(NLastGetopt::TOpts* parser, NCatboostOptions::T
 void BindPoolLoadParams(NLastGetopt::TOpts* parser, NCatboostOptions::TPoolLoadParams* loadParamsPtr) {
     BindQuantizerPoolLoadParams(parser, loadParamsPtr);
 
+    parser->AddLongOption("init-model", "Init model path")
```

Comment:
looks good

could you please add a test to catboost/pytest to check that train + init-model = train + baseline, where baseline = init-model(train data)?

could you please also add a paragraph about init-model somewhere in catboost/doc/en?

Reply:
Are you talking about the `--baseline-model-snapshot` option? Maybe this is enough and I don't need a new option.

Reply:
https://catboost.ai/en/docs/concepts/input-data_baseline